docs/specification/current.md

```diff
@@ -455,7 +456,7 @@ Below is an example `fhirAuthorization` parameter:
     "access_token": "some-opaque-fhir-access-token",
     "token_type": "Bearer",
     "expires_in": 300,
-    "scope": "patient/Patient.read patient/Observation.read",
+    "scope": "user/Patient.read user/Observation.read",
     "subject": "cds-service4"
   }
```

Comment:
Since this is directly in the authorization section, let's include the new parameter in the example to recommend its population:

```suggestion
    "scope": "patient/Patient.read patient/Observation.read",
    "subject": "cds-service4",
    "patient": "1288992"
  }
```